linux/Jenkinsfile

```diff
@@ -224,11 +278,15 @@ def jenkinsStepDeb() {
         if ("${DebARCH}" == 'armv7l') {
             debLabel = 'armv8&&build&&dockerBuild&&dockerhost'
         }
-
+        if ("${DebARCH}" == 'riscv64') {
+            debLabel = 'riscv64&&dockerInstaller'
+        }
         // reallocate jenkins agent per element in list
         node("linux&&${debLabel}") {
             setup('Debian', "${DebARCH}")
-            tool name: 'jfrog-cli', type: 'jfrog'
+            if ("${DebARCH}" != 'riscv64') {
```

Comment:
Is this logic correct, other logic above has riscv with the jfrog cli

Reply:
Yes, the new way this works is to do the package build on a risc-v machine, publish those packages to jenkins, and then upload them from an x64 machine, as the jfrog-cli currently does not support riscv, so its a change to the process for riscv.. that said, if jfrog ever add riscv support to the jfrog cli, we'll be able to realign this functionality.